test/framework/logger.go

```diff
@@ -77,9 +77,10 @@ func getLogger(name string, options *Opts) Logger {
 	// implementing the logr.Logger interface. This logger will
 	// be propagated through the whole operator, generating
 	// uniform and structured logs.
-	logger := createLogger(options)
-	//logger.Logger = logf.Log.WithName(name)
-	logger.Logger.WithName(name)
+	logger := Logger{
+		createLogger(options).WithName(name).WithValues("namespace", name),
```

Comment:
testing it and using only 
```suggestion
		createLogger(options).WithName(name),
```
seems to be enough, `logger` value is now appearing

```
{"level":"info","T":"2021-01-20T09:03:13.060+0100","logger":"kogito-operator-system","msg":"Execute command","command":"oc","args":["apply","-f","/tmp/kogito-operator339882841.yaml"]}
{"level":"info","T":"2021-01-20T09:03:20.137+0100","logger":"kogito-operator-system","msg":"Wait 5m0s for Kogito operator running"}
{"level":"info","T":"2021-01-20T09:03:23.400+0100","logger":"kogito-operator-system","msg":"'Kogito operator running' is successful"}
{"level":"info","T":"2021-01-20T09:03:23.427+0100","logger":"tradisso-local-cucumber-3813","msg":"Scenario Deploy process-quarkus-example with Maven profile persistence,events with events using Kogito Runtime"}
{"level":"info","T":"2021-01-20T09:03:23.428+0100","logger":"tradisso-local-cucumber-695b","msg":"Scenario Deploy process-quarkus-example with Maven profile persistence,events with events using external Kafka"}
{"level":"info","T":"2021-01-20T09:03:23.431+0100","logger":"tradisso-local-cucumber-72b5","msg":"Scenario Deploy process-quarkus-example with Maven profile native,persistence,events with events using external Kafka"}
{"level":"info","T":"2021-01-20T09:03:23.431+0100","logger":"tradisso-local-cucumber-695b","msg":"Step","stepText":"Namespace is created"}
{"level":"info","T":"2021-01-20T09:03:23.431+0100","logger":"tradisso-local-cucumber-3813","msg":"Step","stepText":"Namespace is created"}
{"level":"info","T":"2021-01-20T09:03:23.432+0100","logger":"tradisso-local-cucumber-695b","msg":"Creating namespace","namespace":"tradisso-local-cucumber-695b"}
{"level":"info","T":"2021-01-20T09:03:23.432+0100","logger":"tradisso-local-cucumber-72b5","msg":"Step","stepText":"Namespace is created"}
{"level":"info","T":"2021-01-20T09:03:23.433+0100","logger":"tradisso-local-cucumber-72b5","msg":"Creating namespace","namespace":"tradisso-local-cucumber-72b5"}
```

Reply:
Yeah earlier we were setting the logger value but not storing it again in the, the withName was acting on value rather than on reference :)